docs/software/apps-and-envs/lammps.md

```diff
@@ -115,6 +114,8 @@ The following script illustrates how to run the LAMMPS binary built with the KOK
 
 module load lammps/21Nov2023
 
+ulimit -l unlimited
+
 cd $SLURM_SUBMIT_DIR
 
 ntasks_per_node=$SLURM_NTASKS_PER_NODE
```

Comment:
- Batch scripts:
   - "!/bin/bash" => "#!/bin/bash"
   - "-input" => "-in"
- Last batch script fails with/out ulimit line.
- First two batch scripts run fine with/out ulimit line.

Reply:
Thanks!